### DISCUSSION TITLE
Instructions to Render gt table Quarto/Markdown to MS Word

### DISCUSSION THREAD

**Discussion by twelvespot** [rstudio/gt#1161] [Q&A]
Hi there,

I've tried to follow the example from release 7 and the gtcars dataset to render a report to Word.  However, the table doesn't show up in the render.  Am I doing something wrong?  I show an example here where I call the table at the end of a code chunk AND I try to call the table inline with single backticks.

`---
title: "gt cars practice"
format: docx
editor: visual
---

## Quarto

```{r}
#| include: false
library(tidyverse)
library(gt)
```

Here's a gt table for gtcars

```{r}
#| echo: false
# Use `gtcars` to create a gt table;
# add a header and then export as
# OOXML code for Word
tab_rtf <-
  gtcars %>%
  dplyr::select(mfr, model) %>%
  dplyr::slice(1:2) %>%
  gt() %>%
  tab_header(
    title = md("Data listing from **gtcars**"),
    subtitle = md("`gtcars` is an R dataset")
  ) |> 
    as_word()
tab_rtf
```

Here should be the table too
`r tab_rtf`
`
The screenshot below is what is rendered.

Any help, please!
<img width="1289" alt="Screenshot 2022-12-06 at 7 20 36 PM" src="https://user-images.githubusercontent.com/54380538/206055986-c3ccfe1e-6e94-4101-811b-b42ce42d1607.png">
<img width="623" alt="Screenshot 2022-12-06 at 7 21 02 PM" src="https://user-images.githubusercontent.com/54380538/206056246-8e9ed901-16d8-447a-8a0b-d6cd9e41d69d.png">

**Comment by James-G-Hill** [comment]
I've been able to get `gt::gt()` to render a table to Word within a Quarto document without the `as_word` function. However, this only worked on a Windows 11 machine for me. When I deployed my application to a Ubuntu server the rendering failed and output the xml-type code just as in your screenshot above.

**Comment by AaronGullickson** [comment]
Try using the yaml header:

```yaml
prefer-html: true
```

And then just print the `gt` as usual out of the code chunk without `as_word`. Word basically will try to read the html code to make the table.The resulting table in word often needs some post-rendering formatting to get everything looking right, but it will all be there.

**Comment by renatovargas** [comment]
What's the point of `as_word()` if you can't use the output for anything? I've tried using it with `writeClipboard(as_word(gt_tbl))` and when I paste to Word, I just get the raw OOXML pasted there. I can do `writeClipboard(as_html(gt_tbl))`, paste to Excel and then paste back into word. Of course, this is when not working with quarto or md, which will render just fine out of the box, without adding `as_anything()`. But for quick outputs, I wish I could just paste the OOXML output in Word and you'd get a table. What's the point of `as_word()` then, how does anyone use it?
